pkg/controller/signup.go

```diff
@@ -28,7 +28,7 @@ func NewSignup(config *configuration.Registry, signupService signup.Service) *Si
 
 // PostHandler creates a Signup resource
 func (s *Signup) PostHandler(ctx *gin.Context) {
-	userSignup, err := s.signupService.CreateUserSignup(ctx.GetString(context.UsernameKey), ctx.GetString(context.SubKey), ctx.GetString(context.EmailKey))
+	userSignup, err := s.signupService.CreateUserSignup(ctx.GetString(context.UsernameKey), ctx.GetString(context.SubKey), ctx.GetString(context.EmailKey), ctx.GetString(context.GivenName), ctx.GetString(context.FamilyNameKey), ctx.GetString(context.Company))
```

Comment:
There are too many params...
Let's maybe just pass the context as the only param and extract the required properties from the context in the service?

Reply:
Agreed, fixed as part of: 809fe7555398f3cfc60debeeb9f8c8cab16ec423